launcher.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/sh
```

Comment:
Shouldn't it be part of repo with source code?

Reply:
Yes, but as before in a future release.